config/ocp-observatorium-test.yaml

```diff
@@ -87,57 +109,69 @@ scenarios:
             allpanics: 'sum(rate({msg=~"panic.*"}[1m]))'
             countpanics: 'count_over_time({msg=~"panic.*"}[5m])'
             topTenErrors: 'topk(10, sum(rate({component!="", level="error"}[5m])) by (component))'
-          startThreshold: 1024000
+          args:
+            destination: loki
+            log-lines-rate: 10
   highVolumeWrites:
     enabled: true
     configurations:
       - description: "Write 1,000 logs-per-second"
         samples:
-          interval: "20s"
-          range: "30s"
-          total: 18
+          interval: "30s"
+          range: "300s"
+          total: 10
         writers:
           replicas: 1
+          command: generate
           args:
             destination: loki
             log-lines-rate: 1000
+            synthetic-payload-size: 100
```

Comment:
We do not need to set `synthetic-payload-size` unless specify also `source` as `synthetic` (this will change the way logs are crested by the logger into synthetic logs with defined length). The default are simple log lines